resources/views/admin/blocks/tours_grid.blade.php

```diff
@@ -0,0 +1,10 @@
+@twillBlockTitle('Tours Grid')
+@twillBlockIcon('image')
+
+@formField('browser', [
+'routePrefix' => 'visit',
+'moduleName' => 'customToursLandingPageItems',
+'name' => 'customToursLandingPageItems',
+'label' => 'Tours',
+'max' => 6,
```

Comment:
Please remove this limit

Reply:
Thanks, I've renamed all occurrences of "Tours" to "Custom Tour", with the exception of the page_type which is now Pre-made Tours to avoid clashes with the existing Custom page_type. 

I've also set the max to -1 (unlimited), as removing it stops you being able to select multiple custom tours at once.